src/units/pathfinder2remaster/player-core/ancestry/dwarf/heritage/heritage-remaster-dwarf-rock-dwarf.yml

```diff
@@ -0,0 +1,21 @@
+unit: heritage/dwarf/remaster/rock
+in: heritage/dwarf
+group: "_{Player Core}"
+name: "_{Rock Dwarf}"
+
+inc:
+  - set: heritage
+    value: "_{Rock Dwarf}"
+
+  - at: '@heritage'
+    replace:
+      - article:
+        title: "_{Rock Dwarf}"
+        contents:
+          - p: "_{+2 to saving throw DCs against attempts to move, reposition, shove, trip or knock you prone.}"
+          - p: "_{Any effect that would move you 10ft or more only moves you half as far.}"
+
+  - at: '@saving-throws'
+    add:
+      - p: "_{+2 to DCs against attempts to move, reposition, shove, trip or knock you prone.}"
```

Comment:
If you can use the same string both times you describe the rule, it'll mean less work for translators.

When a bonus says eg +2, I like to use `value-block` to make the number stand out.

```
  - value-block: "+2"
    content: "_{DCs against attempts to move, reposition, shove, trip or knock you prone.}"
```